### PR TITLE
[zh-cn]: create the translation of `aria-brailleroledescription`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20821

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
请先修正一下标点符号

File: files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<article
+  aria-roledescription="slide"
+  aria-brailleroledescription="sld"
+  aria-labelledby="slide1heading">
+  <h1 id="slide1heading">Welcome to my talk</h1>
+  <img alt="Me" src="images/me.jpg" />
+</article>
+```
+
+大多数辅助技术，如屏幕阅读器，将以上内容读作 "slide，welcome to my talk。Image，Me。"。盲文辅助技术将在盲文中呈现 "sld welcome to my talk gra me"。语义 {{HTMLElement('article')}} 通过 `aria-roledescription` 属性被赋予 "slide" 角色；"slide" 是规范中未定义的角色，但是是演示文稿中幻灯片的常见角色。在盲文中，角色呈现为 "sld"。"gra" 是 "graphic" 的简称，在盲文中表示 "image" 角色的缩写。
```
Comment (tianyeeT) [inline files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:30]:
```suggestion
大多数辅助技术，如屏幕阅读器，将以上内容读作“slide, welcome to my talk. Image, Me.”。盲文辅助技术将在盲文中呈现“sld welcome to my talk gra me”。语义 {{HTMLElement('article')}} 通过 `aria-roledescription` 属性被赋予“slide”角色；“slide”是规范中未定义的角色，但是是演示文稿中幻灯片的常见角色。在盲文中，角色呈现为“sld”。“gra”是“graphic”的简称，在盲文中表示“image”角色的缩写。
```

File: files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  sourceCommit: 019ca5c9ce641bfa02825e1ba0444f35dfb646cc
+---
+
+{{AccessibilitySidebar}}
+
+全局 `aria-brailleroledescription` 属性定义了一个人类可读的、作者本地化的缩写描述，用于指定一个元素的角色，旨在转换为盲文。
+
+## 描述
+
+盲文不是字母和数字的一对一转译，而是包括各种缩写、省略和代表单词的字符（称为词符）。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:14]:
```suggestion
盲文不是字母和数字的一对一转译，而是包括各种缩写、缩略词和代表单词的字符（称为词符）。
```

File: files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  aria-brailleroledescription="sld"
+  aria-labelledby="slide1heading">
+  <h1 id="slide1heading">Welcome to my talk</h1>
+  <img alt="Me" src="images/me.jpg" />
+</article>
+```
+
+大多数辅助技术，如屏幕阅读器，将以上内容读作“slide, welcome to my talk. Image, Me.”。盲文辅助技术将在盲文中呈现“sld welcome to my talk gra me”。语义 {{HTMLElement('article')}} 通过 `aria-roledescription` 属性被赋予“slide”角色；“slide”是规范中未定义的角色，但是是演示文稿中幻灯片的常见角色。在盲文中，角色呈现为“sld”。“gra”是“graphic”的简称，在盲文中表示“image”角色的缩写。
+
+`aria-brailleroledescription` 属性应仅用于澄清非交互式容器角色（如 "group" 或 "region"）的目的，或者在盲文环境中提供小部件的更具体描述。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:32]:
```suggestion
`aria-brailleroledescription` 属性应仅用于澄清非交互式容器角色（如“group”或“region”）的目的，或者在盲文环境中提供微件的更具体描述。
```

File: files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- 避免使用 Unicode 盲文模式。如果必须使用，请确保 `aria-brailleroledescription` 值包含除 Unicode 盲文模式、空白字符和盲文模式点 0 之外的内容。
+- 确保该值始终针对文档的语言进行本地化。
+
+> **警告：** 如果内容仅为 Unicode 盲文模式，则该值将不会根据用户的首选翻译表进行翻译。
+
+> **备注：** 不要使用 `aria-brailleroledescription` 复制 `aria-roledescription`。只有当 `aria-roledescription` 无法提供足够的盲文表示时才包含此属性。
+
+如果 `aria-brailleroledescription` 的值满足以下情况，将不会向盲文用户显示：
+
+- 值为空，或仅包含空白字符或空的盲文模式：点 -0（U+2800）。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:51]:
```suggestion
- 值为空，或仅包含空白字符或空的盲文模式：点 0（U+2800）。
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Comment by tianyeeT** [comment]
Thanks! @yin1999

**Review by yin1999 — APPROVED** [review]

**Pull request merged**